core/iwasm/include/wasm_export.h

```diff
@@ -222,6 +222,10 @@ typedef struct RuntimeInitArgs {
 #define LOAD_ARGS_OPTION_DEFINED
 typedef struct LoadArgs {
     char *name;
+    /* This option is only used by the Wasm C API (see wasm_c_api.h) */
+    bool clone_wasm_binary;
+    /* This option is only used by the Wasm C API (see wasm_c_api.h) */
```

Comment:
Should be used by AOT/wasm loader only. Had better add detailed comment here, and in wasm-c-api.h: `This option is only used by the AOT/wasm loader (see wasm_export.h)`